src/main/java/ch/ivyteam/ivy/maven/SetupIvyTestPropertiesMojo.java

```diff
@@ -91,6 +92,27 @@ private void setIvyProperties(MavenProperties properties)
     {
       properties.setMavenProperty(Property.IVY_PROJECT_IAR_CLASSPATH, getClasspath(iarCp));
     }
+    
+    File tstVmDir = createTestVmRuntime();
+    properties.setMavenProperty(Property.IVY_TEST_VM_RUNTIME, tstVmDir.getAbsolutePath());
+  }
+
+  private File createTestVmRuntime() throws MojoExecutionException
+  {
+    IvyTestRuntime testRuntime = new IvyTestRuntime();
+    testRuntime.setProductDir(identifyAndGetEngineDirectory());
+    try
+    {
+      File target = new File(project.getBuild().getOutputDirectory()).getParentFile();
+      File tstVmDir = new File(target, "ivyTestVm");
```

Comment:
I would move this code into the store method and let the store method return the file that needs to be set on the classpath. In this way it is more similar to what we have in the core.

Reply:
I decided against this :  I'll have to start passing around project vars and let the IvyTestVmRuntime make aware of /target folder structures. 
like this I can keep it a damn simple POJO. And the gluing code is in the mojo - which feels natural in maven code.

Reply:
looked again on the code.. think I'll give a try to send the project as parameter to the store method.

Reply:
Simple provide the project File to the pojo. So that the pojo do not have to know the project but is responsible for its own directory structure